src/ui/pages/GetStarted/GetStarted.tsx

```diff
@@ -95,8 +120,69 @@ function Options() {
     '/connect-hardware-wallet',
     beforeCreate
   );
+  const addReadonlyHref = createNextHref('/get-started/readonly', beforeCreate);
   return (
     <Background backgroundKind="white">
```

Comment:
Let's change <Background> to useBackgroundKind here?

Reply:
Yeah ok!